src/lib/fragments/AgGrid.react.js

```diff
@@ -459,7 +459,9 @@ export default class DashAgGrid extends Component {
         if (rowModelType === 'clientSide') {
             const virtualRowData = [];
             this.state.gridApi.forEachNodeAfterFilterAndSort((node) => {
-                virtualRowData.push(node.data);
+                if (node.data) {
```

Comment:
what's the condition where you get falsy `node.data`?

Reply:
If the rows are grouped, they put undefined because the row is only for displaying the data in the grid. 

You can see it if you group rows and print out the virtualRowData.

Reply:
But that doesn't apply to `forEachNode` or `forEachLeafNode`?

Reply:
`forEachNode` it is an issue. Not with `forEachLeafNode`, but neither of these options take into account the sort or filter of the grid. So it has to be tested for.